### PR TITLE
init add of Xamarin rules

### PR DESCRIPTION
**Pull request by mike-hunhoff**
Adding new rules based on #706 . This PR may be impacted by the discussions in #818 regarding mobile ATT&CK techniques. I wanted to open a PR for discussion before diving too far into creating new rules.

Questions:

- Should Android example files be placed in separate tests folder?

### CODE REVIEW

File: nursery/access-camera-in-dotnet-on-android.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: access camera in .NET on Android
+    namespace: host-interaction/hardware/camera
+    authors:
+      - michael.hunhoff@mandiant.com
+    scope: function
+  features:
+    - or:
+      - api: Android.Hardware.Camera::Open
```
Comment (williballenthin) [inline nursery/access-camera-in-dotnet-on-android.yml:10]: restrict to `os: android`? or maybe the name and API namespace are sufficient?

we'll have to come up with a consistent style on things like this. i don't know today.

File: nursery/get-system-information-in-dotnet-on-android.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: get system information in .NET on Android
```
Comment (williballenthin) [inline nursery/get-system-information-in-dotnet-on-android.yml:3]: seems rather generic. could we make this name a little more specific? is it the OS version or similar?

**Review by williballenthin — APPROVED** [review]
so cool to see

File: nursery/access-camera-in-dotnet-on-android.yml (hunk 1 above)
Comment (mr-tz) [reply nursery/access-camera-in-dotnet-on-android.yml:10]: os and/or format restrictions make sense to add

**Review by mr-tz — COMMENTED** [review]
I think the testfiles should be separate.

Similarly, do we want to keep the rules files separate? Some rules contain various formats/OS so that's a little messy.
There's pro and cons, so maybe we better open up a separate discussion?

File: nursery/access-camera-in-dotnet-on-android.yml (hunk 1 above)
Comment (mike-hunhoff) [reply nursery/access-camera-in-dotnet-on-android.yml:10]: Discussed offline w/ @mr-tz . We'll leave this for now and continue discussion in the issue that I just created to track this.

**Pull request merged**